config/config.go

```diff
@@ -445,6 +445,9 @@ type RPCConfig struct {
 	// Maximum size of request body, in bytes
 	MaxBodyBytes int64 `mapstructure:"max_body_bytes"`
 
+	// Maximum number of requests in a request body
+	MaxRequestBatchRequest int `mapstructure:"max_request_batch_request"`
```

Comment:
Sorry, I think the name of `max_request_batch_request` seems a bit odd now, because the `request` is added twice. 😅

Reply:
How about `max_batch_request_size` ?

Reply:
`max_batch_request_num`?